cmd/ipfs/init.go

```diff
@@ -73,20 +75,23 @@ environment variable:
 		return nil
 	},
 	Run: func(req cmds.Request, res cmds.Response) {
+		// needs to be called at least once
+		res.SetOutput(nil)
```

Comment:
i know i've asked this before, but remind me again why this is?

Reply:
Because `res.Output` is called concurrently and we need to make sure `SetOutput` is called before `Output` reads the value. So there is a channel read at the beginning of `Output`, and a channel close (guarded by a `sync.Once`) at the end of `SetOutput`.
So if we didn't call `SetError`, the channel wouldn't close and `Output` would sit there and wait forever.